benchmarks/scripts/run_accuracy.py

```diff
@@ -219,8 +262,12 @@ def main(args):
     parser.add_argument("--torch_npu_version", type=str, required=False)
     parser.add_argument("--vllm_version", type=str, required=False)
     parser.add_argument("--cann_version", type=str, required=False)
+    parser.add_argument("--vllm_commit", type=lambda s: s[:7], required=False)
```

Comment:
only `vllm_commit` and `vllm_ascend_commit` are needed, other var can be generate from them